sample_usage/print_table.py

```diff
@@ -0,0 +1,63 @@
+"""
+Examples of how to use the AccumulationTable Class with its
```

Comment:
You can remove the phrase "with its respective outputs", I'm not sure exactly what you mean. The main block is good and speaks for itself.